data/templates/Detector_Automation.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+		| where TIMESTAMP == startTime and TIMESTAMP == endTime
```

Comment:
| where TIMESTAMP >= startTime and TIMESTAMP <= endTime

Reply:
Done, however I'll start charging hourly from the next change :P